pyiron_atomistics/lammps/base.py

```diff
@@ -393,13 +387,32 @@ def write_input(self):
         self._set_selective_dynamics()
         if update_input_hdf5:
             self.input.to_hdf(self._hdf5)
-        self.input.control.write_file(
-            file_name="control.inp", cwd=self.working_directory
-        )
-        self.input.potential.write_file(
-            file_name="potential.inp", cwd=self.working_directory
-        )
-        self.input.potential.copy_pot_files(self.working_directory)
+        if self.input.potential.files is not None:
+            files_to_copy = {os.path.basename(f): f for f in self.input.potential.files}
+        else:
+            files_to_copy = {}
+        return {
+            "files_to_create": {
+                "structure.inp": lmp_structure._string_input,
+                "control.inp": "".join(self.input.control.get_string_lst()),
+                "potential.inp": "".join(self.input.potential.get_string_lst()),
+            },
+            "files_to_copy": files_to_copy,
+        }
+
+    def write_input(self):
```

Comment:
Shouldn't this go to `AtomisticGenericJob`?

Reply:
Yes, the corresponding pull request is already open on `pyiron_base` https://github.com/pyiron/pyiron_base/pull/1472 .

Reply:
Then why is a copy here?

Reply:
This is primarily for historic reasons, this was the first pull request, before the one in `pyiron_base`. The copy is removed in https://github.com/pyiron/pyiron_atomistics/pull/1451

Reply:
But doesn't it make more sense to not add it in the first place in this PR?

Reply:
Unless a new version of base is released the tests fail without this temporary function.

Reply:
I'm ok with it, but it seems to me like a bigger evil to copy and paste lines of code only to be deleted in the next PR if the sole reason to do it is you want to merge it earlier, because it's gonna stay in the commit history and might confuse the future developers. Maybe someone like @pmrv or @liamhuber has an opinion?

Reply:
@samwaseda, what you are saying sounds perfectly reasonable to me. If this PR ought to depend on a PR in base, unless there is some pressing need to get it merged and released, just wait for what's needed to be available in base.